index.html

```diff
@@ -1,71 +1,83 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Curriculum Vitae</title>
-  <style>
-    img {
-      margin-left: 25px;
-        }
-    main {
-      margin-left: auto;
-      margin-right: auto;
-      padding: 25px;
-      width: 1024px;
-      border: solid 1px;
-      padding: 25px;             
-        }
-  </style>
-</head>
-<body>
-  <main>
-    <header>        
-      <h1 align="center">Curriculum Vitae</h1>            
-      <img src="avatar.jpg", alt="Здесь должна быть аватарка" 
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Curriculum Vitae</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <div class="header-name">
+        <h1>Kiseliov Stanislav</h1>
+        <h2 class="gradient">Junior Front-End Developer</h2>
+        <div class="contacts">
+          <ul>
+            <li class="icon">
+              <img src="icon/icon-phone.png" alt="" width="20" height="20"/>
+              <a href="tel:+375293135198"> +375293135198 </a>
+            </li>
+            <li class="icon">
+              <img src="icon/icon-mail.png" alt="" width="20" height="20"/>
+              <a href="mailto:stif@tut.by"> stif@tut.by</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="icon">
+              <img src="icon/icon-github.png" alt="" width="20" height="20"/>
+              <a href="https://github.com/KiStanislav"> kistanislav</a>
+            </li>
+            <li class="icon">
+              <img src="icon/icon-linkedin.png" alt="" width="20" height="20"/>
+              <a href="https://www.linkedin.com/in/stanislav-kiseliov-b93923158">linkedin.com/stanislav</a>
```

Comment:
Please, add hover style for links. It is a common practice nowadays.

Reply:
Done!